source/languages/en/riak/dev/advanced/write-once.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Write Once Buckets
+project: riak
+version: 2.1.0+
+document: tutorial
+audience: intermediate
+keywords: [developers, buckets]
+---
+
+Riak 2.1.0 introduces the concept of "write once" buckets; buckets whose entries
```

Comment:
Can we change this to:
"Riak 2.1.0 introduces the concept of Write Once Buckets, buckets whose entries are intended to be written exactly once and never updated or overwritten."